CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # Changelog
 
+## 4.0.1 / X June 2021
+
+- implement interstial ad slot option
+
 ## 4.0.0 / 19 May 2021
 
-- Use of Prebid.js is now optional, you can display ads using only GPT (issue
-  #37, PR #44)
+- Use of Prebid.js is now optional, you can display ads using only GPT (issue #37, PR #44)
```

Comment:
Please use Prettier to format Markdown files, that will wrap prose at 80 characters and unintended changes like this won't happen anymore